README.md

```diff
@@ -60,6 +60,12 @@ and prefixes in class names, so not `CurrenciesInterface` but `Currencies`. Othe
 using the same code style as we are using. Please provide tests when creating a PR and clear descriptions of bugs when
 filing issues.
 
+You can run the following commands to get started:
+```
+composer install
```

Comment:
The terminal sign (`$`)? Yes, that's why I didn't add it at first place. I'll add it though. Makes more sense.